deepcave/plugins/hyperparameter/importances.py

```diff
@@ -178,6 +179,26 @@ def process(run, inputs):
         if n_trees is None:
             raise RuntimeError("Please specify the number of trees.")
 
+        # Handle constant values in fANOVA: As the fANOVA implementation relies on pyrfr and pyrfr cannot be applied
+        # to constant hyperparameters (see https://github.com/automl/fanova/issues/81), as a workaround we remove
+        # constant hyperparameters before calculation.
+        # Note: This will break if there are conditions or forbiddens including constant hyperparameters.
```

Comment:
Do we have that documented in the documentation? If not, please add that there as well.

Reply:
Added to the docs.